### ISSUE TITLE
LinqBridge package is still referenced when targeting .NET 3.5 or later

### ISSUE THREAD

**Issue by alser** [ua-parser/uap-csharp#53]
Package version 3.1.40

Steps to reproduce:

1. Create empty .net framework console application, select target .NET 3.5
2. Install package UAParser 3.1.40
3. Use any linq extensions in Program.cs, for example:

```
using System.Collections.Generic;
using System.Linq;

namespace ConsoleApp1
{
    class Program
    {
        static void Main(string[] args)
        {
            var items = new List<int>().FirstOrDefault();
        }
    }
}
```

4. Build the project

> error CS0121: The call is ambiguous between the following methods or properties: 'System.Linq.Enumerable.FirstOrDefault<TSource>(System.Collections.Generic.IEnumerable<TSource>)' and 'System.Linq.Enumerable.FirstOrDefault<TSource>(System.Collections.Generic.IEnumerable<TSource>)'

Exactly same issue occurs when targeting .net 4.7.2 or other frameworks with embedded Linq support.

Please, consider adding "net35" target platform without LinqBridge in addition to netstandard`s so that NuGet can select it instead of net20

**Comment by simonqbs** [comment]
I also got the same error, so I ended up using 3.1.38 (although it did fill up my packages.config pretty bad). I guess #52 would fix this?

**Comment by enemaerke** [comment]
I've merge #52 and published 3.1.41 just now. Let me know if that fixes things for you guys

**Comment by alser** [comment]
problem solved in 3.1.41, thanks!
